builder/qemu/step_run.go

```diff
@@ -16,41 +16,45 @@ import (
 // stepRun runs the virtual machine
 type stepRun struct {
 	DiskImage bool
-}
 
-type qemuArgsTemplateData struct {
```

Comment:
Moved this template data to live next to the interpolation it's used to perform.